src/ontology/mondo-edit.obo

```diff
@@ -71,6 +71,14 @@ property_value: IAO:0000700 MONDO:0021125
 property_value: IAO:0000700 MONDO:0021178
 property_value: IAO:0000700 MONDO:0042489
 
+[Term]
+id: MONDO:0000000
+name: TRIP11 related skeletal dysplasia
+def: "TRIP11 related skeletal dysplasia describes both achondrogenesis type 1a and ondontochondrodysplasia. Reduced protein function in TRIP11 causes a spectrum of skeletal symptoms from a more mild phenotype, called ondontochondrodysplasia, with features including short stature and joint laxity to a more severe phenotype, called achondrogenesis type 1a, which presents as a lethal prenatal or neonatal skeletal dysplasia. The clinical severity of achondrogenesis compared to ondontochondrodysplasia is related to the residual function of the gene which is not currently possible to anticipate based on genotype alone. The phenotype cannot fully be predicted by genotype alone, evidenced by variants being reported with both phenotypes." [PMID:20089971, PMID:30728324, PMID:31903676]
```

Comment:
For the definition, the definitions should start with the parent class and not mention the disease label, because they can change. This disease should also follow this pattern: https://github.com/monarch-initiative/mondo/blob/master/src/patterns/dosdp-patterns/disease_series_by_gene.yaml

I suggest revising the definition to:

Any skeletal dysplasia in which the cause of the disease is a variation in the TRIP11 gene.  Reduced protein function in TRIP11 causes a spectrum of skeletal symptoms from a more mild phenotype, called ondontochondrodysplasia, with features including short stature and joint laxity to a more severe phenotype, called achondrogenesis type 1A, which presents as a lethal prenatal or neonatal skeletal dysplasia. The clinical severity of achondrogenesis compared to ondontochondrodysplasia is related to the residual function of the gene which is not currently possible to anticipate based on genotype alone. The phenotype cannot fully be predicted by genotype alone, evidenced by variants being reported with both phenotypes." [PMID:20089971, PMID:30728324, PMID:31903676]

Reply:
Note, I don't think it is necessary to mention "TRIP11 related skeletal dysplasia describes both achondrogenesis type 1a and ondontochondrodysplasia." because these are subclasses and that is implied.